restangular/restangular.d.ts

```diff
@@ -116,6 +116,9 @@ declare namespace restangular {
     withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
     save(queryParams?: any, headers?: any): IPromise<any>;
     getRestangularUrl(): string;
+    route?: string;
+    id?: string;
```

Comment:
It depends on the application you're working with. Often times its number
but with several Microsoft applications such as SharePoint and Dynamics CRM
the id is actually a string formed GUID. Maybe using type of any would be
better here to accommodate more endpoints?

On Mon, May 23, 2016, 6:45 AM Blaz <notifications@github.com> wrote:

> In restangular/restangular.d.ts
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9240#discussion_r64202149>
> :
>
> > @@ -116,6 +116,9 @@ declare namespace restangular {
> >      withHttpConfig(httpConfig: angular.IRequestShortcutConfig): IElement;
> >      save(queryParams?: any, headers?: any): IPromise<any>;
> >      getRestangularUrl(): string;
> > +    route?: string;
> > +    id?: string;
>
> @vvakame <https://github.com/vvakame> is string correct type for id,
> shouldn't this be number ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/DefinitelyTyped/DefinitelyTyped/pull/9240/files/83dcde9de2c6259aeafab1c0c68929bea0f95a2d#r64202149>
>